WooCommerce/Classes/System/DefaultFeatureFlagService.swift

```diff
@@ -11,6 +11,8 @@ struct DefaultFeatureFlagService: FeatureFlagService {
             return true
         case .shippingLabelsRelease2:
             return buildConfig == .localDeveloper || buildConfig == .alpha
+        case .shippingLabelsM4:
```

Comment:
nit: it's probably better to rename it to `shippingLabelsRelease4` or, better, use this new naming convention also for release 2 (`shippingLabelsM2M3` or something like that. It will become super clear that the feature flag 2 includes also the functionalities under M3)

Reply:
Good point! Updated to use `shippingLabelsM2M3` for release 2 in 354af60.